api/tests/routes/pro/get_venues_of_offerer_from_siret.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+
+import pcapi.core.offerers.factories as offerers_factories
+from pcapi.core.testing import assert_num_queries
+
+from tests.conftest import TestClient
+
+
+@pytest.mark.usefixtures("db_session")
+class Returns200Test:
+    def test_get_venues_of_offerer_from_siret(self, app):
+        siren = "123123123"
+        offerer = offerers_factories.OffererFactory(siren=siren)
+        user_offerer = offerers_factories.UserOffererFactory(user__email="user.pro@test.com", offerer=offerer)
+
+        permanent_venue = offerers_factories.VenueFactory(
+            name="Permanent", managingOfferer=user_offerer.offerer, siret=f"{siren}12323", isPermanent=True
+        )
+
+        offerers_factories.VenueFactory(
+            name="Not Permanent",
+            managingOfferer=user_offerer.offerer,
+            isPermanent=False,
+            siret=f"{siren}12324",
+        )
+
+        auth_request = TestClient(app.test_client()).with_session_auth(email=user_offerer.user.email)
```

Comment:
(style) On ne devrait plus utiliser `TestClient` ainsi. La fixture pytest `client` doit être utilisée à la place. Il suffit de l'ajouter aux arguments de la fonction de test (ici `client` va remplacer `app` dans la liste des arguments), et ça s'utilise comme suit : 

```suggestion
        auth_request = client.with_session_auth(email=user_offerer.user.email)
```